extensions/wikia/AdEngine/js/provider/evolve2.js

```diff
@@ -17,7 +17,6 @@ define('ext.wikia.adEngine.provider.evolve2', [
 			'320x50,320x100,300x250': 'a',
 			'300x250': 'a'
 		},
-		section = evolveHelper.getSect(),
```

Comment:
It is used by previous evolve provider:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/evolve.js#L32

We'll be able to remove whole evolveHelper once we decide to move on Evolve2